cobigen/index.asciidoc

```diff
@@ -1,28 +1,30 @@
-= CobiGen Cli
+= Tutorial for Java Backend Generation with CobiGen
 ====
-CobiGen is a generic incremental code generator. It allows you build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
+CobiGen is a generic incremental code generator. It allows you to build Java CRUD application based on the devonfw architecture including all software layers. You can generate all necessary classes and services (DAOs, Transfer Objects, simple CRUD use cases with REST services) of the application from one single entity class.
 
 CobiGen provides Maven and Eclipse integrations. But there is also a command line interface (CLI) which enables the generation of code using only few commands. In this scenario, you will learn how to install and use the cobigen cli inside a java project.
 
 More information about CobiGen on https://devonfw.com/website/pages/docs/master-cobigen.asciidoc.html
 ====
 
-To use the cobigen cli you first have to install the devonfw ide. You will find more information about devonfw on https://devonfw.com/website/pages/welcome/welcome.html.
 [step]
 --
-installDevonfwIde(["java","mvn"])
+restoreDevonfwIde(["java","mvn"])
 --
 
-Now, you have to download cobigen
+For this tutorial an installation of the Devon IDE is required.
```

Comment:
Here you can mention that the installation was already done and the user can use the IDE right away